docs/_rtd/virtualenv.md

```diff
@@ -1,3 +1,28 @@
 ## Setting Up Your Virtual Environment
 
-Add information about how to set up Anaconda.
+If you wish to do higher-level development on top of `ark`, we recommend setting up a virtual environment. We highly recommend using `conda` virtual environments. To be able ot set one up, you will need to install the Anaconda package.
+
+### Installing Anaconda
+
+For a step-by-step guide of how to install Anaconda, please refer to these links:
+* https://docs.anaconda.com/anaconda/install/mac-os/ for Mac users
+* https://docs.anaconda.com/anaconda/install/windows/ for Windows users
+
+We recommend using the graphical installer for ease of use.
+
+### Creating a virtual environment
+
+Now that Anaconda is installed, you can now create a `conda` environment. 
+ 
+To do so, on your command line, type `conda create -n <my_env> python=3.6`, where `<my_env>` is a name you set. We highly recommend setting `python=3.6` because this is the version of Python our code uses. 
```

Comment:
Instead of highly recommending it, we should use stronger language.  Maybe something more like: **'Do not change the python version.  Our codebase assumes python 3.6.'**

Optionally, you could also include something like: **'If your contribution requires a later version of python, post an issue to the github requesting the codebase be updated to python x.y, listing appropriate reasonings and rationales for updating.'**

Reply:
> Looks good overall. One extra thing here is that there should be a section for contributing to ark via virtual env. Right now, seems like everything in here is focused on using ark as a package.

@ackagel I feel like most of the contributing guidelines are already handled in `contributing.md`. Is there anything specific to contributing related to `conda` environments that we need to put in `virtualenv.md`? If so, we might want to consider putting that stuff in `contributing.md` instead of `virtualenv.md` instead.

Reply:
`contributing.md` doesn't currently mention much with regards to setting up your environment for development (be it `conda`, `virtualenv`, `pipenv`, `docker`, `vagrant`, etc).  It mostly touches on preferred etiquette, and shies away from technical details.

So, since having some technical details would be good, I think linking to the `virtualenv.md` document from `contributing.md` when mentioning environments could be a good, along with having a section in `virtualenv.md` concerning how you would start contributing to ark within a conda env.

Definitely open to other suggestions, but I think this one keeps `contributing.md` clean/non-technical, while still providing a more technical contribution resource.

Reply:
Agree with both of you. We don't want any contributing relating details in here. This should basically be a how-to guide for setting a virtual-env for someone who is going to be developing the code, as opposed to just building the docker and running the notebooks